docs/user-guide/creating-application/workflow/cd-pipeline.md

```diff
@@ -84,7 +84,20 @@ Make sure your cluster has `devtron-agent` installed if you check the `Execute i
 
 ![](../../../.gitbook/assets/cd_pre_build%20%282%29.jpg)
 
-### 4. Deployment stage
+### 4. Manual approval for deployment
+
+When `manual approval for deployment` is enabled, only approved images are available for deployment through the respective deployment pipeline.
+Users can specify the number of approvals required for each deployment, with a minimum of 1 and a maximum of 6.
```

Comment:
Change this to
Users can specify the number of approvals required for each deployment, where the allowed limit is minimum 1 approval or maximum 6 approval.